Userland/Libraries/LibGfx/ImageFormats/TGALoader.h

```diff
@@ -6,28 +6,66 @@
 
 #pragma once
 
+#include <AK/MemoryStream.h>
 #include <LibGfx/ImageFormats/ImageDecoder.h>
 
 namespace Gfx {
 
-struct TGALoadingContext;
+enum TGADataType : u8 {
```

Comment:
No need to leak implementation details in the Gfx namespace, a forward declaration is fine